### ISSUE TITLE
Capitalization in "updated" string and limitation of the current translation method

### ISSUE THREAD

**Issue by Ede123** [JasonBarnabe/greasyfork#353]
Code in question:
1. https://github.com/JasonBarnabe/greasyfork/blob/HEAD/app/helpers/application_helper.rb#L22
2. https://github.com/JasonBarnabe/greasyfork/blob/HEAD/config/locales/en.yml#L16 (translation)

**There are two issues:**

**First** the code in 1. capitalizes the String. This might not be wanted in all languages. Some of the current translations do not use `%{relative_time}` at the start of the translated string.

**Second** The translation provided through [rails-i18n](https://github.com/svenfuchs/rails-i18n) is very limited.
E.g. for German it [outputs something](https://github.com/svenfuchs/rails-i18n/blob/HEAD/rails/locale/de.yml#L81) like `"ein Tag"` or `"zwei Tage"`.
A correct German translation of the whole string in question would be `"Vor einem Tag"` or `"Vor zwei Tagen"` respectively. As you can see the Grammar does not work out.
Only workaround that would be compatible with both - rails and capitalization - would be `"Ein Tag zuvor"` or `"Zwei Tage zuvor"` respectively. (It's not really nice and would never be used in spoken German but it works).

**In conclusion** I suggest either of two approaches to solve this issue:
1. Make translators aware, that `%{relative_time}` is supposed to be used in the beginning of the translation and live with the limitations.
2. Find an alternative approach that produces nicer translations than what rails-i18n seems to be able to do.

**Comment by JasonBarnabe** [comment]
If I understand you correctly, the text in German is based on the context (here the context is referring to a time something happened in the past), and the context rails-18n is using is not correct here. I imagine this is an issue in other languages as well, because even in English it's kind of kludgy. I believe this issue was raised in https://github.com/svenfuchs/rails-i18n/issues/217, and the answer is to not use that helper in this case.

I think I will include translations for minutes/hours/days ago then they can be correct for what we're doing.

**Comment by Ede123** [comment]
Yes, it's basically the same issue.

Having our own translations with proper plural form would solve the issue indeed, if this is a workable solution for you.

**Comment by JasonBarnabe** [comment]
It's workable because we don't need them all. We can skip the about/less than/more than, as well as seconds/months/years. So just the translations for minutes/hours/days, and their plural forms.

**Comment by JasonBarnabe** [comment]
Is the German nicer now?

**Comment by Ede123** [comment]
I surely hope so ( ...since it was me who translated it on Transifex ;) ).

I assume it's flexible enough for all languages now, thanks!
